src/nvim/tui/term_input.inl

```diff
@@ -262,7 +263,7 @@ static void read_cb(RStream *rstream, RBuffer *buf, void *data, bool eof)
         break;
       }
     }
-  } while (rbuffer_size(input->read_buffer));
+  } while (rbuffer_size(input->read_stream.buffer));
 
   // Make sure the next input escape sequence fits into the ring buffer
   // without wrap around, otherwise it could be misinterpreted.
```

Comment:
`rbuffer_reset(input->read_buffer);` needs to be replaced with `rbuffer_reset(input->read_stream.buffer);` in the following line.